lib/auth/helpers.go

```diff
@@ -902,6 +902,23 @@ type clt interface {
 	UpsertUser(types.User) error
 }
 
+// CreateRole creates a role without assigning any users. Used in tests.
+func CreateRole(clt clt, name string, spec types.RoleSpecV4) (types.Role, error) {
+	ctx := context.TODO()
```

Comment:
```suggestion
func CreateRole(ctx context.Context, clt clt, name string, spec types.RoleSpecV4) (types.Role, error) {
```

Reply:
That's reasonable, I've made the change.